website/pages/en/quick-start.mdx

```diff
@@ -35,133 +30,141 @@ Using [yarn](https://yarnpkg.com/):
 yarn global add @graphprotocol/graph-cli
 ```
 
-## 3. Initialize your subgraph from existing contract
+### 2. Create your subgraph
+
+If your contract has events, the `init` command will automatically create a scaffold of a subgraph.
+
+#### Create via CLI
+
+Use the following command to create a subgraph in Subgraph Studio using CLI: `init --product subgraph-studio`
+
+#### **Create via Subgraph Studio**
+
+Subgraph Studio is a staging environment where you can create, manage, deploy, and publish subgraphs and API Keys.
```

Comment:
Just a nitpick, I'm not sure it's technically correct to say that Studio _is_ a staging environment. I guess it _has_ a staging environment, but it is mostly a dapp.